test/test.hs

```diff
@@ -291,6 +295,28 @@ testHoughLinesP = do
     lineSegments <- houghLinesP 1 (pi / 180) 100 Nothing Nothing edgeImgM
     assertBool "no lines found" (V.length lineSegments > 0)
 
+testNewCascadeClassifierAlgorithm :: HU.Assertion
+testNewCascadeClassifierAlgorithm = do
+  mbCC <- newCascadeClassifier "/this/is/bogus.xml"
+  case mbCC of
+    Nothing -> return ()
+    Just _cc -> fail "expected Nothing from newCascadeClassifier"
+
+testCascadeClassifierDetectMultiScaleArnold :: HU.Assertion
+testCascadeClassifierDetectMultiScaleArnold = do
+  Just ccFrontal <- newCascadeClassifier "/usr/local/share/OpenCV/haarcascades/haarcascade_frontalface_default.xml"
```

Comment:
Well, one thing we can do is to just include the `xml` file inside the package, if the license allows it.